avalanche-network-runner-sdk/src/lib.rs

```diff
@@ -27,6 +28,11 @@ pub struct GrpcClient<T> {
     pub control_client: Mutex<ControlServiceClient<T>>,
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct GlobalConfig {
+    pub log_level: String,
```

Comment:
Q. Does this work with underscore? Or should we use log-level for json tag?

Reply:
will add a test and fix 0:)